src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -180,7 +180,7 @@ private void testCopyOptions() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("CONTRIB");
         feedbackEditPage.fillQuestionTextBoxForNewQuestion("q2, contribution qn");
-        assertEquals("Please select a visibility option",
```

Comment:
Exactly what the test checks for now - `Please select a visibility option`. You can check out the current behaviour by seeing what happens when you add a new (non-contrib) question to an empty session.